docs/apache-airflow/security/webserver.rst

```diff
@@ -85,9 +85,41 @@ Please use command line interface ``airflow users create`` to create accounts, o
 Other Methods
 '''''''''''''
 
-Standing on the shoulder of underlying framework Flask-AppBuilder, Airflow also supports authentication methods like
-OAuth, OpenID, LDAP, REMOTE_USER. You can configure in ``$AIRFLOW_HOME/webserver_config.py``.
-For details, please refer to
+Since the Airflow 2.0, the default UI is the Flask App Builder RBAC. A ``webserver_config.py`` configuration file
+it's automatically generated and can be used to configure the Airflow to support authentication
+methods like OAuth, OpenID, LDAP, REMOTE_USER.
+
+For previous versions from Airflow, the ``$AIRFLOW_HOME/airflow.cfg`` following entry needs to be set to enable
+the Flask App Builder RBAC UI.
+
+.. code-block:: ini
+
+    rbac = True
+
+The default authentication option is the AUTH_DB, and a new user can be created through a CLI command or by the
+UI modifying the ``$AIRFLOW_HOME/webserver_config.py`` with the following options:
```

Comment:
I have the impression that this sentence is not completely correct. Why should the user edit this file? To add a new user?

Reply:
Edit this file as the code example I wrote will allow user self registration.
I will edit this section to make it clear